### ISSUE TITLE
[FEATURE] music player takes lots of space even when no music is available

### ISSUE THREAD

**Issue by LacombeLouis** [TheBoredTeam/boring.notch#214]
**Is your feature request related to a problem? Please describe.**
When no music is playing in the background, the music app still takes up about two-thirds of the space on the notch. I think it would be better if it either didn’t show up at all or had a cleaner display. The album art and pause/skip buttons feel quite unnecessary in this context.

**Describe the solution you'd like**
Not entirely sure what would be the ideal output solution.

**Additional context**
<img width="561" alt="Screenshot 2024-11-19 at 11 27 42" src="https://github.com/user-attachments/assets/14ef5358-db42-44d4-80af-9712f6e052a4">

**Checks**
- [x] I haven't found any duplicates with my issue.

**Comment by Alegzob** [comment]
Yes I totally agree, I'd also say that it often doesn't match what's actually playing, for example I have Spotify opened in the background but watching a video on YouTube, and the paused song will show up instead of the YouTube video, sometimes the opposite will happen.
